darwin/dataset/local_dataset.py

```diff
@@ -169,6 +173,93 @@ def parse_json(self, index: int):
             "annotations": annotations,
         }
 
+    def measure_mean_std(self, multi_threaded: bool = True):
```

Comment:
I saw this function used elsewhere, if it's exactly the same let's make sure we are smart with code duplication

Reply:
I think you saw that function in the class `DarwinDataset` in `darwin.torch.dataset`, which has now become `LocalDataset` in `darwin.dataset`. I'll double check anyways.